artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: etcdproxy-clusterrole
+rules:
```

Comment:
my mental model for this controller says that it will need full CRUD on secrets inside of this namespace to create the required certs.